Formula/f/flint.rb

```diff
@@ -21,23 +21,31 @@ class Flint < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "796f363a97eb42772140b94d31a9eaccfb2acf445175639aba7869250b293942"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "gmp"
   depends_on "mpfr"
   depends_on "ntl"
-
   uses_from_macos "m4" => :build
 
   def install
+    # to build against NTL
     ENV.cxx11
+    # Allow -march options to be passed through, as FLINT builds
+    # arch-specific code with runtime detection of capabilities:
+    # https://flintlib.org/doc/building.html
+    ENV.runtime_cpu_detection
+
+    system "./bootstrap.sh" if build.head?
+
     args = %W[
       --with-gmp=#{Formula["gmp"].prefix}
       --with-mpfr=#{Formula["mpfr"].prefix}
       --with-ntl=#{Formula["ntl"].prefix}
```

Comment:
FLINT does not need NTL to install `NTL-interface.h`. Removing `--with-ntl` may reduce build-time and may reduce the complexity for installing.

However, you may still want FLINT to depend on NTL if you want to download NTL when downloading FLINT. `--with-ntl` is only used for checking, but perhaps there is some `make check` here I didn't see?

Reply:
there is no make check. So I will remove the `--with-ntl`, as long it still installs the header

Reply:
I have confirmed that the headers are still being installed.